README.md

```diff
@@ -135,7 +138,8 @@ There are 5 ways to install Gogs:
 
 ## Contributors
 
-- Ex-team members [@lunny](https://github.com/lunny), [@fuxiaohei](https://github.com/fuxiaohei) and [@slene](https://github.com/slene).
+- Owners [@lunny](https://github.com/lunny), [@tboerger](https://github.com/tboerger) and [@bkcsoft](https://github.com/bkcsoft).
```

Comment:
I would not mention that here, we can skip the maintainance of this part. You can see that anyway on the people page of the org.